test/context.test.ts

```diff
@@ -7,22 +7,22 @@ import type { WebhookDefinition } from "@octokit/webhooks-examples";
 import fetchMock from "fetch-mock";
 import { describe, expect, test, beforeEach, it, vi } from "vitest";
 
-import { Context } from "../src";
-import { ProbotOctokit } from "../src/octokit/probot-octokit";
+import { Context } from "../src/index.js";
+import { ProbotOctokit } from "../src/octokit/probot-octokit.js";
 import type { PushEvent } from "@octokit/webhooks-types";
 
 const pushEventPayload = (
-  WebhookExamples.filter(
```

Comment:
What's going on here? Doesn't typescript pick up the `index.d.ts`?

Reply:
It doesn't when ESM mode is enabled. Probably because the main file of @octokit/webhook-types is in JSON, and JSON imports in ESM aren't supported yet. They work at runtime, because Node.js has backwards-compatibility with non-ESM packages.

I assume this is a bug in TypeScript.